front/admin_ui/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gradio==~3.16.1
```

Comment:
I think we should have only one: pyproject.toml or requirements.txt

Reply:
I added requirements.txt to easily deploy it on Spaces - I don't think Spaces support pyproject.toml files.

Reply:
ok!
If needed, we could later generate the file automatically in the CI with https://python-poetry.org/docs/cli/#export, in order to avoid duplicating the dependencies information.